Documentation/troubleshooting.md

```diff
@@ -2,11 +2,11 @@
 
 ## Firewall
 
-Running DHCP or proxyDHCP with `coreos/dnsmasq` on a host requires that the Firewall allow DHCP and TFTP (for chainloading) services to run.
+Running DHCP or proxyDHCP with `coreos/dnsmasq` on a host requires that the firewall allows the DHCP and TFTP (for chainloading) services to run.
 
-## Port Collision
+## Port collision
 
-Running DHCP or proxyDHCP can cause port already in use collisions depending on what's running. Fedora runs bootp listening on udp/67 for example. Find the service using the port.
+Running DHCP or proxyDHCP can cause port already in use collisions depending on what's running. Fedora runs bootp listening on udp/67 for example. Find the service that uses the port.
```

Comment:
changed because "using" can have ambiguous meaning -> "by using the port" and "that which uses the port"